test/NuGet.Core.Tests/NuGet.Commands.Test/MSBuildRestoreUtilityTests.cs

```diff
@@ -3399,6 +3400,27 @@ public void MSBuildRestoreUtility_AddPackageDownloads_InvalidVersionRange(string
             var exception = Assert.Throws<ArgumentException>(() => MSBuildRestoreUtility.AddPackageDownloads(spec, msbuildItems));
         }
 
+        [Fact]
+        public void MSBuildRestoreUtility_AddPackageDownloads_NoVersion_ThrowsException()
+        {
+            // Arrange
+            PackageSpec spec = MSBuildRestoreUtility.GetPackageSpec(new[] { CreateItems(new Dictionary<string, string>()) });
+            var packageX = new Mock<IMSBuildItem>();
+            const string packageId = "x";
```

Comment:
Good idea making this `const`, I don't always remember to do that when writing tests.